components/collection/activity/holdersFlippersTabs/moreNFTS.vue

```diff
@@ -0,0 +1,27 @@
+<template>
```

Comment:
Please use more suitable name for the component 🤗. 
Other components seems to be named correctly

Reply:
renamed to NFTSDetailsDropdown